pkg/idhubmain/commitroot.go

```diff
@@ -0,0 +1,38 @@
+package idhubmain
+
+import (
+	"github.com/ethereum/go-ethereum"
+	log "github.com/golang/glog"
+	"github.com/joincivil/id-hub/pkg/claims"
+)
+
+// RunCommitRoot whatever the current root is for the root_merkletree is saved to the smart contract
+func RunCommitRoot() error {
+	config := populateConfig()
```

Comment:
There wouldn't be a problem if this script runs multiple times in a row correct?

Reply:
I think it would just waste some eth